OrbitCore/Path.cpp

```diff
@@ -258,42 +258,16 @@ std::string Path::GetAppDataPath() {
 
 std::string Path::GetMainDrive() { return GetEnvVar("SystemDrive"); }
 
-std::string Path::GetSourceRoot() {
-  std::string currentDir = GetExecutablePath();
-  const int numIterations = 10;
-  const std::string fileName = "bootstrap-orbit";
-
-  for (int i = 0; i < numIterations; ++i) {
-    if (ContainsFile(currentDir, fileName)) {
-      return currentDir;
-    }
-
-    currentDir = GetParentDirectory(currentDir);
-  }
-
-  return "";
-}
-
-std::string Path::GetExternalPath() {
-  if (is_packaged_)
-    return GetExecutablePath() + "external/";
-  else
-    return GetSourceRoot() + "external/";
+static std::string GetSourceRoot() {
+  std::string currentDir = Path::GetDirectory(__FILE__);
+  return Path::GetParentDirectory(currentDir);
```

Comment:
This assumes that Path.cpp is only one level below the source root, right?

Reply:
Wasn't the previous implementation trying to be clever, although still relying on the very uncertain assumption that "bootstrab-orbit" be in the source root?

Reply:
yes - added comment about it.